### PR TITLE
WorldHub: Make worlds directory path configurable

### PR DESCRIPTION
**Pull request by aexer0e**
# WorldHub: Configurable Worlds Directory Path

Added support for setting the directory path WorldHub tries to fetch worlds from within the project's `config.json` so that we're no longer constricted to just "worlds/". 

## Usage
To set a custom directory path, add a `"WorldHub"` entry in `config.json`, within which set the `"dirPath"` key to the value of your custom worlds directory, like so:
```
{
    "WorldHub": {
        "dirPath": "my-worlds"
    }
}
```

## Backwards Compatibility
When un-set, WorldHub will default to using "worlds".

### CODE REVIEW

**Comment by solvedDev** [comment]
Some functionality inside of bridge. (e.g. packaging) currently still relies on the "worlds/" folder (See #389). Since we've seen the need to customize the folder, we've already engaged on the project config standard to add a new "worlds" property to the config file: https://github.com/Bedrock-OSS/project-config-standard/issues/2

I am open to merge the PR for now though

**Comment by aexer0e** [comment]
I see. I think the standardized "worlds" property within a project's config should be implemented and parsed within dash itself and be available for extensions/custom compilers as a hook since it's fairly sophisticated and that way 3rd party utilizations won't have to worry about how to go about parsing project worlds, but until then I think this is a good compromise and it wouldn't be harmful if a standardization is implemented at a lower level in the future.

**Comment by solvedDev** [comment]
> I think the standardized "worlds" property within a project's config should be implemented and parsed within dash or bridge.?

Yes, this parsing of this property is already implemented for a couple of bridge. versions and can be accessed over the project config module. (There's a new, undocumented `config.getWorldHandles(): Promise<FileSystemDirectoryHandle[]>` method on the ProjectConfig class)

File: plugins/WorldHub/scripts/worldHub.js [hunk 1]
```diff
@@ -3,15 +3,22 @@ const { create } = await require('@bridge/sidebar')
...
+} = await require('@bridge/fs')
 const { getCurrentProject } = await require('@bridge/env')
+const Config = await readJSON(`${getCurrentProject()}/config.json`)
 
 class WorldHubTab extends ContentTab {
 	type = 'WorldHubTab'
 	component = WorldHub
 	availableWorlds = null
 	isReady = false
+	worldsDir = Config?.WorldHub?.dirPath || 'worlds'
```
Comment (solvedDev) [inline plugins/WorldHub/scripts/worldHub.js:21]: I would prefer the "WorldHub" namespace property to start with a lowercase letter to align it with the other properties within the project config -> Use "worldHub" instead

**Review by solvedDev — CHANGES_REQUESTED** [review]
Generally looks good to me. Please adjust the property name and then this should be ready to merge

**Review by solvedDev — APPROVED** [review]
LGTM

**Pull request merged**